src/platform/native.zig

```diff
@@ -316,10 +316,16 @@ pub const Platform = struct {
             .double => .fifo,
             .triple => .mailbox,
         };
+
+        platform.last_position = try platform.window.getPos();
+
         if (options.fullscreen) {
-            platform.last_position = try platform.window.getPos();
 
-            const monitor = glfw.Monitor.getPrimary().?;
```

Comment:
Thanks for sending this change. Happy to merge this.

However, I don't think the default `monitor: u32 = 0` is always going to be equal to `glfw.Monitor.getPrimary()`. I would like for the default to remain the primary monitor.

Can you make `monitor: u32 = 0` instead be `monitor: ?u32 = null` and choose the `getPrimary()` monitor if null?